lib/chromedriver.js

```diff
@@ -16,11 +16,11 @@ const DEFAULT_HOST = '127.0.0.1';
 const DEFAULT_PORT = 9515;
 class Chromedriver extends events.EventEmitter {
   constructor (args = {}) {
```

Comment:
args can be defined as `{host, port, executable, cmdArgs, adb, verbose, logPath}` right here